### PR TITLE
usbus: add usbus_add_interface_alt() helper function

### PR DESCRIPTION
**Pull request by dylad**
### Contribution description

This PR add a helper function to easily add one (or more) `usbus_interface_alt_t` to a given interface.
The alternate settings is not that much used in USBUS yet so there is no direct impact for now.
However, having multiple alternate settings for a given interface can be useful in some applications (like USB DFU).

### Testing procedure
We don't use more than one alternate settings at this time so we cannot test this with our current codebase.
CDC ECM use one alternate setting, this should still work with this PR.

### Issues/PRs references
None

### CODE REVIEW

File: sys/usb/usbus/usbus.c [hunk 1]
```diff
@@ -121,6 +121,17 @@ uint16_t usbus_add_interface(usbus_t *usbus, usbus_interface_t *iface)
     return idx;
 }
 
+void usbus_add_interface_alt(usbus_t *usbus, usbus_interface_t *iface,
```
Comment (bergzand) [inline sys/usb/usbus/usbus.c:124]: Is there a reason for including `usbus_t` in the parameters here? It is not used in the function.

Comment (dylad) [reply sys/usb/usbus/usbus.c:124]: There is no specific reason. I just copy-paste the add_interface fonction and change the bare minimum.
Should I drop this parameter ?

**Comment by dylad** [comment]
Parameter dropped.
And PR squashed at the same time.

**Review by bergzand — APPROVED** [review]
Ack, tested with the DFU PR

**Pull request merged**